ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -145,4 +145,36 @@ public void checkIfRightPatient() {
             confirmButton2.click();
         }
     }
+    
+    public void clearPatientHeight() {
```

Comment:
Thanks @kdaud , Where are these HEIGHT_FIELD  element references defined?

Reply:
> Where are these HEIGHT_FIELD element references defined?

Defined at `Code LN 13` within the same file.